lib/scholar/cluster/k_means.ex

```diff
@@ -239,9 +280,23 @@ defmodule Scholar.Cluster.KMeans do
   end
 
   @doc """
-  Makes predictions with the given model on inputs `x`.
+  Makes predictions with the given `model` on inputs `x`.
+
+  ## Returns
```

Comment:
How about standardizing on "Return Values" or "Results" instead of "Returns"?

Reply:
"Return Values" is cool 👍